### DISCUSSION TITLE
Facing issue while setting up the new workspace

### DISCUSSION THREAD

**Discussion by Dinesh0723** [eclipse-platform/eclipse.platform#1170] [Q&A]
I am encountering the error 'org.eclipse.equinox.p2.director code=10054 Cannot complete the install because of a conflicting dependency' while setting up the new workspace. Has anyone else experienced this issue?

**Comment by merks** [comment]
How can I reproduce this issue?

**Comment by Dinesh0723** [comment]
Steps to Reproduce:

1. Download and execute the Oomph installer.exe, then switch to advanced mode.
2. Select the product (Eclipse IDE for Committers) and choose the latest version from the product version combo box.
3. Click on the Network Proxy Settings icon and add the necessary proxy details.
4. Choose the Eclipse projects from the checkbox tree viewer (e.g.: Platform).
5. Select the installation folder, then click "Next" and subsequently "Finish".

During installation, the process is interrupted, and I encounter the error 'org.eclipse.equinox.p2.director code=10054 Cannot complete the install because of a conflicting dependency'."

**Comment by merks** [comment]
You haven't mentioned which version of the Eclipse IDE for Committers you picked.  This works for me:

![image](https://github.com/eclipse-platform/eclipse.platform/assets/208716/12475b94-b9bf-404c-a237-6c8dab66edd3)

Probably your best to use the configuration setup:

https://github.com/eclipse-platform/.github/blob/main/CONTRIBUTING.md#creating-an-eclipse-development-environment

You can always go back to the projects page and deselect projects.

You can probably also avoid the problem by using Platform SDK product:

![image](https://github.com/eclipse-platform/eclipse.platform/assets/208716/ad58c856-a453-443d-80d0-f34dcc36557c)
